index.bs

```diff
@@ -25,6 +25,12 @@ urlPrefix: https://tc39.github.io/ecma262/; spec: ECMA-262
   type: dfn; text: realm; url: realm
 urlPrefix: https://storage.spec.whatwg.org/; spec: storage
   type: dfn; text: storage; url: site-storage
+urlPrefix: https://fs.spec.whatwg.org/; spec: fs
```

Comment:
these should not be needed, but you might need to add `export` tags to these definitions in the fs spec.